ElementX/Sources/Other/HTMLParsing/DTHTMLElement+AttributedStringBuilder.swift

```diff
@@ -64,13 +64,11 @@ public extension DTHTMLElement {
         }
     }
     
-    private static var allowedHTMLTags = {
-        ["font", // custom to matrix for IRC-style font coloring
-         "del", // for markdown
-         "body", // added internally by DTCoreText
-         "mx-reply",
-         "h1", "h2", "h3", "h4", "h5", "h6", "blockquote", "p", "a", "ul", "ol",
-         "nl", "li", "b", "i", "u", "strong", "em", "strike", "code", "hr", "br", "div",
-         "table", "thead", "caption", "tbody", "tr", "th", "td", "pre"]
-    }()
+    private static var allowedHTMLTags = ["font", // custom to matrix for IRC-style font coloring
```

Comment:
Good spot. I think this can be a `let` too?
```suggestion
    private static let allowedHTMLTags = ["font", // custom to matrix for IRC-style font coloring
```

Reply:
Thanks, all SwiftFormat's doing 😅